tests/react/use-all-docs.test.tsx

```
@@ -0,0 +1,174 @@
+import { renderHook, waitFor } from "@testing-library/react";
+import { describe, expect, it, beforeEach, afterEach } from "vitest";
+import { fireproof, useFireproof } from "use-fireproof";
+import type { Database } from "use-fireproof";
+import type { AllDocsResult } from "../../src/react/types.js";
+import type { DocWithId } from "@fireproof/core";
```

ESLint

[error] 'DocWithId' is defined but never used.

(@typescript-eslint/no-unused-vars)